src/Geno/C++/Auxiliary/ImGuiAux.cpp

```diff
@@ -24,15 +24,9 @@
 
 //////////////////////////////////////////////////////////////////////////
 
-bool ImGuiAux::RenameTree( std::string& rNameToRename )
+void ImGuiAux::RenameTree( std::string& rNameToRename, bool& rRename, const std::function< bool( void ) >& rCallback )
```

Comment:
Can't this also just return a bool, like it did before? Or is there some other reason as to why this was changed?